ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/mainSample.cpp

```diff
@@ -595,6 +598,8 @@ void registerClasses()
 
   qmlRegisterSingletonType<DownloadSampleManager>("Esri.ArcGISRuntimeSamples", 1, 0, "SampleManager",
                                                   &esriSampleManagerProvider);
+  // Register the Google Analytics class
+  qmlRegisterSingletonType<GAnalytics>("Telemetry", 0, 1, "GAnalytics", &gAnalyticsProvider);
```

Comment:
❓ Why is the version 0.1? I would've thought this would be 1.0 unless it is being considered as beta/alpha/tech preview or something along those lines. As far as I can tell, it is fully functional and is the 1.0 version.